subprojects/core/src/integTest/groovy/org/gradle/caching/internal/packaging/BuildCacheEntryPackingIntegrationTest.groovy

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright 2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.caching.internal.packaging
+
+import org.gradle.integtests.fixtures.DirectoryBuildCacheFixture
+import org.gradle.integtests.fixtures.daemon.DaemonIntegrationSpec
+import spock.lang.Issue
+import spock.lang.Unroll
+
+class BuildCacheEntryPackingIntegrationTest extends DaemonIntegrationSpec implements DirectoryBuildCacheFixture {
+
+    private static final NON_ASCII_NAME = [
```

Comment:
❓ is there any advantage in having privates in test files?

Reply:
I don't think there's anything specific to tests here. This is a value that only should be used in this class, so it should be `private`.

Reply:
Minor: Are there cases when we're referencing fields from other tests? Don't we always treat test fields as implicitly private?

Reply:
I'm not sure what "implicitly private" means. What is the benefit of not declaring this as `private`? Why wouldn't we want to be explicit when something is shared between tests?

Reply:
If we are practicing that, I.e. define a constant in one test and use it in the other, it may make sense to signal that this constant is not one of those.
Since I assume we're not advocating that, we can already assume that everything is private, so explicitly declaring it has little advantage.
I don't mind, just want to clarify.

Reply:
OK, I think I understand your argument now. You are saying that `private` adds only noise here, as we'd never actually reference this field from another test. Do I get that correctly? We do use this inconsistently, and I think it's fine either way.

Reply:
Thanks for taking the time.